specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2243 @@
+      body: AtlasClassification[];
+    },
+    void,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/request-body-problem" "This is Atlas API behavior"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Update classification on an entity identified by its type and unique attributes.")
+  @route("/atlas/v2/entity/uniqueAttribute/type/{typeName}/classifications")
+  @put
+  updateClassificationsByUniqueAttribute is Azure.Core.Foundations.Operation<
+    {
+      @doc("The name of the type.")
+      @path
+      typeName: string;
+
+      @doc("""
+The qualified name of the entity. (This is only an example. qualifiedName can
+be changed to other unique attributes)
+""")
+      @query("attr:qualifiedName")
+      attribute?: string;
+
+      @doc("An array of classification to be updated.")
+      @body
+      body: AtlasClassification[];
+    },
+    void,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Set classifications on entities in bulk.")
+  @route("/atlas/v2/entity/bulk/setClassifications")
+  @post
+  bulkSetClassifications is Azure.Core.Foundations.Operation<
+    {
+      @doc("Atlas entity headers.")
+      @body
+      body: AtlasEntityHeaders;
+    },
+    OkResponse<string[]>,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "This is Atlas API behavior"
+  @doc("""
+Bulk API to retrieve list of entities identified by its unique attributes.
+In
+addition to the typeName path parameter, attribute key-value pair(s) can be
+provided in the following
+format
+
+typeName=\\<typeName>&attr_1:\\<attrName>=\\<attrValue>&attr_2:\\<attrName>=\\<attrValue>&attr_3:\\<attrName>=\\<attrValue>
+
+NOTE:
+The attrName should be an unique attribute for the given entity-type.
+The REST
+request would look something like this
+
+GET
+/v2/entity/bulk/uniqueAttribute/type/hive_db?attr_1:qualifiedName=db1@cl1&attr_2:qualifiedName=db2@cl1
+
+Note:
+at least one unique attribute must be provided.
+""")
+  @route("/atlas/v2/entity/bulk/uniqueAttribute/type/{typeName}")
+  @get
+  listByUniqueAttributes is Azure.Core.Foundations.Operation<
+    {
+      @doc("The name of the type.")
+      @path
+      typeName: string;
+
+      @doc("Whether to return minimal information for referred entities.")
+      @query
+      minExtInfo?: boolean;
+
+      @doc("Whether to ignore relationship attributes.")
+      @query
+      ignoreRelationships?: boolean;
+
+      @doc("""
+Qualified name of an entity. E.g. to find 2 entities you can set
+attrs_1:qualifiedName=db1@cl1&attrs_2:qualifiedName=db2@cl1. (This is only an
+example. qualifiedName can be changed to other unique attributes)
+""")
+      @query
+      `attr_N:qualifiedName`?: string;
```

Comment:
Just a heads up this will probably not work as expected in the SDKs, I understand there are limitations in describing this API, how was this handled in the previous SDKs?

Reply:
In previous SDK, only qualifiedName is supported. Users cannot add multiple unique attributes.

Reply:
The attr_N prefix if valid for the service?

Reply:
Yes. It is valid. It is also an Atlas API. We need to keep compatible